tsdb/tsm1/metrics.go

```diff
@@ -25,6 +25,7 @@ func PrometheusCollectors() []prometheus.Collector {
 		collectors = append(collectors, bms.compactionMetrics.PrometheusCollectors()...)
 		collectors = append(collectors, bms.fileMetrics.PrometheusCollectors()...)
 		collectors = append(collectors, bms.cacheMetrics.PrometheusCollectors()...)
+		collectors = append(collectors, bms.readMetrics.PrometheusCollectors()...)
```

Comment:
This is the missing piece. There is another declaration for collectors for the `blockMetrics` which I did add but I missed this one.